internal/api/front/perses/src/app/project/dashboard/dashboard-details/dashboard-details.component.html

```diff
@@ -1 +1,33 @@
-<p>dashboard-details works!</p>
+<app-page>
+  <app-page-header>
+    <mat-spinner *ngIf="isLoading"></mat-spinner>
+    <h1 *ngIf="!isLoading">{{dashboard.metadata.name}}</h1>
+  </app-page-header>
+
+  <app-page-content>
+    <mat-spinner *ngIf="isLoading"></mat-spinner>
+    <app-page-sub-content *ngIf="!isLoading">
+      <ng-container *ngFor="let section of dashboard.spec.sections">
```

Comment:
I would put that in a card (`mat-card`). It would highlight a bit better the sections part

Reply:
but if you feel it can be done in another PR, it's fine for me too. Or I could do that during the integration of the variables to align the design let's say.

Reply:
done + few other display improvements :

![image](https://user-images.githubusercontent.com/7058693/120010024-f3449300-bfdc-11eb-9982-bd6f148e6b4f.png)